types.d.ts

```diff
@@ -116,6 +132,14 @@ export interface IFlagsmith<F extends string = string, T extends string = string
      * Initialise the sdk against a particular environment
      */
     init: (config: IInitConfig<F, T>) => Promise<void>;
+    /**
+     * Set evaluation context. Refresh the flags.
```

Comment:
We should add a note in here about it merging

Reply:
Added the `updateContext` interface instead.